packages/core/src/components/annotation/annotation.scss

```diff
@@ -0,0 +1,52 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/units";
+
+@use "~dso-toolkit/src/components/insert";
+@use "~dso-toolkit/src/components/delete";
+
+@use "~dso-toolkit/src/components/insert/insert" as *;
+@use "~dso-toolkit/src/components/delete/delete" as *;
+
+:host {
+  display: block;
+}
+
+:host([wijzigactie="voegtoe"]) {
+  @include insert.root();
+}
+
+:host([wijzigactie="verwijder"]) {
+  @include delete.root();
+}
+
+@include utilities.box-sizing();
+
+.annotation-body {
+  display: flex;
+  gap: units.$u1;
+}
+
+.annotation-info {
+  flex-grow: 1;
+}
+
+.annotation-symbol,
+.annotation-control {
+  flex-shrink: 0;
+}
+
+.annotation-name,
+.annotation-data {
+  margin-block: 0;
+}
+
+.annotation-name {
+  font-weight: 500;
+}
+
+.annotation-data {
+  span:has(+ dso-label),
+  dso-label:has(+ dso-label) {
```

Comment:
@hansgrimm Sorteer jij hier voor op meerdere labels achter elkaar?

Reply:
Wat betreft externe API (hoe componenten zich tot elkaar verhouden) begrijp ik het. Deze stylesheet gaat echter uitsluitend over interne werking en is YAGNI en brengt daarom wat mij betreft alleen maar ruis op de lijn.

Ik heb 'm eruit gehaald. Kan jij 'n review doen? Ik heb het een en ander refactored, maar qua CSS/HTML heel basic